.github/bin/dist-nuget.sh

```diff
@@ -3,7 +3,7 @@
 # Note that this script is intended to be run by GitHub Actions.
 set -e
 
-project=Libplanet
+projects="Libplanet Libplanet.RocksDBStore"
```

Comment:
How about using Bash's proper array:

```suggestion
projects=( "Libplanet" "Libplanet.RocksDBStore" )
```

Reply:
We're using POSIX sh which is not supporting arrays. Would it be better to change to bash instead?

Reply:
I added a commit to use bash array. 1f6aa35cbd66b5095b46bc1014b590c8974a5511